### DISCUSSION TITLE
Mistake in qgsunittypes.h

### DISCUSSION THREAD

**Discussion by pby22555** [NHERI-SimCenter/SimCenterCommon#212] [R2D] [answered]
Mistake in qgsunittypes.h
![微信图片_20240818212558](https://github.com/user-attachments/assets/3a8ece10-7cb1-4d18-9826-75be88b32133)

**Comment by pby22555** [comment, marked as answer]
qgsunittypes.h can not work correctly.

**Reply by fmckenna** [reply]
Again I must ask "Are you using 5.15.2?" .. all my developers are and the compilation runs smoothly .. we also have a Dockerfile which also has no issue with the code

**Comment by pby22555** [comment]
Dear fmckena, thank you very much!! These are my QT version. May I ask if I want to install in win 10, what else should I do? I notice that the document on github is for mac. By the way, may I ask which version of git.exe do you use?
![QT](https://github.com/user-attachments/assets/e8f3c92e-8e72-4849-8c0a-03214e5b653a)
![1](https://github.com/user-attachments/assets/4b247684-eba2-4250-9736-1288add93a2c)
![2](https://github.com/user-attachments/assets/e7d104c9-8c44-495a-9804-cc76c58562e0)
